packages/template-chakra-storefront/src/config/constants.js

```diff
@@ -1,198 +0,0 @@
-/*
```

Comment:
Bonus: we are not using this constant file at all in the source code. We've moved it to template-chakra-storefront/config/constant.js, but haven't deleted this file